### PR TITLE
op_mode: T6909: Move "show monitoring" and "show zebra" to "show monitoring frr"

### PR DESCRIPTION
**Pull request by natali-rs1985**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Migration from an old Vyatta component to vyos-1x, please link to related PR inside obsoleted component
- [x] Other (please describe) move op-mode commands:

## Related Task(s)

* https://vyos.dev/T6909
## Component(s) name

op mode
## Proposed changes

Move ```show monitoring``` and ```show zebra``` to ```show monitoring frr```
## How to test

```
vyos@vyos:~$ show monitoring frr
Possible completions:
  <Enter>               Execute the current command
  zebra                 Show Zebra routing information


vyos@vyos:~$ show monitoring frr
MGMT debugging status:

Zebra debugging status:

RIP debugging status:

RIPng debugging status:

OSPF debugging status:

OSPF6 debugging status:

LDP debugging status:

BGP debugging status:

isis debugging status:

BABEL debugging status

openfabric debugging status:

Staticd debugging status

BFD debugging status:

PIMv6 debugging status
```

## Checklist:

- [x] I have read the [**CONTRIBUTING**](https://github.com/vyos/vyos-1x/blob/current/CONTRIBUTING.md) document
- [x] I have linked this PR to one or more Phabricator Task(s)
- [ ] I have run the components [**SMOKETESTS**](https://github.com/vyos/vyos-1x/tree/current/smoketest/scripts/cli) if applicable
- [x] My commit headlines contain a valid Task id
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

**Review by sever-sever — APPROVED** [review]

File: op-mode-definitions/show-monitoring.xml.in [hunk 1]
```diff
@@ -2,12 +2,68 @@
...
+      <node name="monitoring">
         <properties>
           <help>Show currently monitored services</help>
         </properties>
-        <command>vtysh -c "show debugging"</command>
-      </leafNode>
+        <children>
+          <node name="frr">
+            <properties>
+              <help>Show currently monitored frr services</help>
```
Comment (dmbaturin) [inline op-mode-definitions/show-monitoring.xml.in]:
```suggestion
              <help>Show currently monitored FRR services</help>
```

FRR is an abbreviation and should be capitalized.

File: op-mode-definitions/show-monitoring.xml.in [hunk 2]
```diff
@@ -2,12 +2,68 @@
...
+                <command>${vyos_op_scripts_dir}/vtysh_wrapper.sh "show ${@:4}"</command>
+                <children>
+                  <node name="client">
+                    <properties>
+                      <help>Client information</help>
+                    </properties>
+                    <children>
+                      <node name="summary">
+                        <properties>
+                          <help>Brief Summary</help>
```
Comment (dmbaturin) [inline op-mode-definitions/show-monitoring.xml.in]:
```suggestion
                          <help>Brief summary</help>
```

"Summary" is not the first word in a sentence and should not be capitalized.

File: op-mode-definitions/show-monitoring.xml.in [hunk 3]
```diff
@@ -2,12 +2,68 @@
...
+                    <command>${vyos_op_scripts_dir}/vtysh_wrapper.sh "show ${@:4}"</command>
+                  </node>
+                  <node name="router">
+                    <properties>
+                      <help>Zebra Router Information</help>
+                    </properties>
+                    <children>
+                      <node name="table">
+                        <properties>
+                          <help>Table Information about this Zebra Router</help>
```
Comment (dmbaturin) [inline op-mode-definitions/show-monitoring.xml.in]:
```suggestion
                          <help>Zebra routing table information</help>
```

File: op-mode-definitions/show-monitoring.xml.in [hunk 4]
```diff
@@ -2,12 +2,68 @@
...
+                        </properties>
+                        <children>
+                          <node name="summary">
+                            <properties>
+                              <help>Summary Information</help>
```
Comment (dmbaturin) [inline op-mode-definitions/show-monitoring.xml.in]:
```suggestion
                              <help>Summary information</help>
```

**Review by dmbaturin — DISMISSED** [review]
Looks good, I left a few style suggestions.

File: op-mode-definitions/show-monitoring.xml.in (hunk 3 above)
Comment (c-po) [reply op-mode-definitions/show-monitoring.xml.in]: Fixed

**Review by c-po — APPROVED** [review]

**Pull request merged**